.github/workflows/libc-fullbuild-tests.yml

```diff
@@ -0,0 +1,76 @@
+# This workflow is for pre-commit testing of the LLVM-libc project.
+name: LLVM-libc Pre-commit Fullbuild Tests
+
```

Comment:
Can you add:
```
permissions:
  contents: read
```

To each new file you've added.  This is standard best practices for GitHub actions definitions.

Reply:
In the same level of name? Sure. Adding now